src/__tests__/gameHandlerClass.spec.ts

```diff
@@ -0,0 +1,129 @@
+import { GameHandler } from "../gameHandlerClass";
+import { Player } from "../playerClass";
+import { PokemonFactory } from "../pokemonFactory";
+import { PokemonType } from "../pokemonClass";
+import * as data from "../../assets/poke_data.json";
+
+describe("Test Game Handler class", () => {
+  const pokeData = data.pokemons;
+
+  it("GameHandler.isGameFinished should return false if every player has at least one pokemon", () => {
+    // Given
+    const factory_0 = new PokemonFactory(pokeData);
+    const playerOne_0 = new Player("Wojtek", [
+      factory_0.getPokemonByName("bulbasaur"),
+      factory_0.getPokemonByName("charmander"),
+      factory_0.getPokemonByName("pikachu"),
+    ]);
+    const playerTwo_0 = new Player("Ania", [
+      factory_0.getPokemonByName("squirtle"),
+      factory_0.getPokemonByName("weedle"),
+      factory_0.getPokemonByName("pidgey"),
+    ]);
+
+    // When
+    const isGameFinished_0 = new GameHandler(
+      playerOne_0,
+      playerTwo_0
+    ).isGameFinished();
```

Comment:
GameHandler is part of `given`, in `when` there should be only `isGameFinished()`